Tests/CommandsTests/TestCommandTests.swift

```diff
@@ -221,7 +221,6 @@ struct TestCommandTests {
     }
 
     @Test(
-        .issue("https://github.com/swiftlang/swift-package-manager/issues/8955", relationship: .defect),
```

Comment:
suggestion: This issue is merged.  Can we remove the `withKnownIssue` block to confirm whether the test now always passes?